ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/versions/deneb/BlockContentsSchema.java

```diff
@@ -47,11 +48,18 @@ public BlockContentsSchema(
         namedSchema("block", schemaRegistry.get(BEACON_BLOCK_SCHEMA)),
         namedSchema(
             FIELD_KZG_PROOFS,
-            SszListSchema.create(SszKZGProofSchema.INSTANCE, specConfig.getMaxBlobsPerBlock())),
+            SszListSchema.create(SszKZGProofSchema.INSTANCE, maxBlobsOrCommitments(specConfig))),
```

Comment:
Looks I didn't explain this clearly, sorry. I think we should always use max blob commitments here, see, for example this structure (and any else in spec) https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/validator.md#blobsbundle

Reply:
we currently do for fulu but not for other branches, they're consistent deneb/deneb etc atm

this particular change makes a mess so i was thinking of doing it when we backport all the BPO to deneb/electra, or maybe before, but it breaks tests, removes OpenApiIntegrationTest references etc

Reply:
i changed that in this PR ultimately, will see if it passes AT

Reply:
looks fine - marking resolved